src/content/docs/start-here/getting-started.mdx

```diff
@@ -192,7 +192,7 @@ In the meantime, we will show you how to make calls using grpcurl
 <Tabs syncKey="language">
   <TabItem label="Python">
   ```bash
-  pip install whatever-the-package-is
+  pip install inventory-grpc-client-python-kessel 
```

Comment:
Everything else should be equal, but we are moving to use [kessel-sdk](https://pypi.org/project/kessel-sdk/)
The latest version also supports oauth now!

Probably worth to note that there are [more examples](https://github.com/project-kessel/kessel-sdk-py/tree/main/examples)

```suggestion
  pip install kessel-sdk 
```